pkgs/development/libraries/raylib/default.nix

```diff
@@ -56,18 +56,18 @@ stdenv.mkDerivation (finalAttrs: {
   passthru.tests = [ raylib-games ];
 
   patches = [
-    # Patch version in CMakeList to 4.5.0
+    # Patch version in CMakeList to 5.0.0
     # Remove this when updating to a new revision
```

Comment:
```suggestion
    # Patch version in CMakeLists.txt to 5.0.0
    # Change this when updating to a new revision
```

Reply:
I don't understand why this is better. it's only necessary to update if they forget to add the build revision again in the release

Reply:
If the goal is to prevent people from making the same mistake i did. It's better to having something more descriptive like "please check that the output library matches the build version or update this patch"

Reply:
The devs say they're going to update the version number next release: https://github.com/raysan5/raylib/issues/3639

Just keep the comment as-is, the patch should probably be removed next update.

Reply:
> It's better to having something more descriptive like "please check that the output library matches the build version or update this patch"

Then let's go with that. The repo owner doesn't use cmake, so it's liable to happen again.

Reply:
@eclairevoyant 
> Then let's go with that. The repo owner doesn't use cmake, so it's liable to happen again.
Updated the comment to be more descriptive